redpanda/resources/cluster/schema.go

```diff
@@ -0,0 +1,257 @@
+package cluster
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/listplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/mapplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/redpanda-data/terraform-provider-redpanda/redpanda/validators"
+)
+
+func resourceClusterSchema() schema.Schema {
```

Comment:
Is this generated or hand crafted?

Reply:
Hand crafted. There's unfortunately no up to date generator for TPF (sdkv2 has a couple).